src/pydata_sphinx_theme/assets/styles/content/_quotes.scss

```diff
@@ -2,5 +2,25 @@
 blockquote {
   padding: 0 1em;
   color: var(--pst-color-text-muted);
-  border-left: 0.25em solid var(--pst-color-blockquote-border);
+  border-left: 0.25em solid var(--pst-color-border);
+  border-radius: 0.2rem;
+  position: relative;
+
+  p {
+    color: var(--pst-color-text-muted);
+  }
+
+  // This is a hack to control the opacity for admonitions with our color variables
+  // ref: https://stackoverflow.com/a/56951626/6734243
+  &:after {
```

Comment:
Just a note now that we've got this used in many places in the theme, I think that we could turn this into a `@mixin` if we wanted. Then we could just use `include` to add this behavior. E.g., something like
```scss
@mixin background-from-color-variable($color-variable) {
  // This is a hack to control the opacity for admonitions with our color variables
  // ref: https://stackoverflow.com/a/56951626/6734243
  &:after {
    content: "";
    width: 100%;
    height: 100%;
    position: absolute;
    left: 0;
    top: 0;
    background-color: var(#{$color-variable});
    background-color: var(--pst-color-border);
    opacity: 0.1;
    border-radius: 0.2rem;
  }
}
```

and then call it with something like:

```scss
blockquote {
  ...css rules
  @include background-from-color-variable(--pst-color-border);
}
```

for ref: https://sass-lang.com/documentation/at-rules/mixin